generation/framework/htcondor/HTCondor.py

```diff
@@ -171,6 +194,18 @@ def htcondor_job_config(self, config, job_num, branches):
             )
             config.custom_content.append(("+RemoteJob", self.htcondor_remote_job))
             # config.custom_content.append(("+RequestWalltime", max_runtime))
+        if self.domain == self.Domain.NAF:
+            config.custom_content.append(("+RequestRuntime", max_runtime))  # NAF
+            # NAF has some custom stuff for containers...
+            if self.htcondor_universe == "docker":
+                raise NotImplementedError("Docker via HTCondor is not supported on NAF!")
```

Comment:
Maybe tell what should be used instead in this message?

Reply:
I guess it is '"container"'.

Reply:
Actually, NAF only supports the vanilla universe. The container implementation is using their vanilla wrapper.
But telling the user to switch to container is fine as I implemented all the necessary changes for this.